pom.xml

```diff
@@ -125,31 +125,25 @@ under the License.
 
   <properties>
     <javaVersion>8</javaVersion>
-    <minimalMavenBuildVersion>3.5.4</minimalMavenBuildVersion>
-
-    <sisuVersion>0.9.0.M2</sisuVersion>
-    <classWorldsVersion>2.7.0</classWorldsVersion>
-    <commonsCliVersion>1.5.0</commonsCliVersion>
-    <commonsIoVersion>2.11.0</commonsIoVersion>
-    <commonsLangVersion>3.12.0</commonsLangVersion>
+    <classWorldsVersion>2.8.0</classWorldsVersion>
+    <commonsCliVersion>1.7.0</commonsCliVersion>
+    <commonsIoVersion>2.16.1</commonsIoVersion>
+    <commonsLangVersion>3.14.0</commonsLangVersion>
     <junitVersion>4.13.2</junitVersion>
     <hamcrestVersion>2.2</hamcrestVersion>
     <mockitoVersion>4.11.0</mockitoVersion>
-    <!-- plexus 2.1.1 is broken, no new release exists yet (nor will exist) -->
-    <plexusVersion>2.1.0</plexusVersion>
-    <plexusInterpolationVersion>1.26</plexusInterpolationVersion>
-    <!-- Blocked by 3.4.0+ changes, see MNG-7710 -->
-    <plexusUtilsVersion>3.5.1</plexusUtilsVersion>
-    <guiceVersion>5.1.0</guiceVersion>
-    <guavaVersion>32.0.1-jre</guavaVersion>
-    <guavafailureaccessVersion>1.0.1</guavafailureaccessVersion>
+    <plexusVersion>2.2.0</plexusVersion>
```

Comment:
unsure, where is this used at all? I'd leave it as there was some issue with qdox, that am unsure was fixed or not

Reply:
I've updated qdox for 2.2.0 release. Also, all Plexus annotations are marked as Deprecated - ie more visible.
Not sure if 3.9 would be beneficial as looks like we don't have plans to do any Plexus -> jsr330 migration.